controller/config.go

```diff
@@ -450,6 +454,10 @@ const (
 	// By default we only want to trace 10% of the requests.
 	DefaultOpenTelemetrySampleRatio = 0.1
 
+	// DefaultOpenTelemetryTailSamplingThreshold is the default value for the
+	// tail sampling threshold for open telemetry.
+	DefaultOpenTelemetryTailSamplingThreshold = 1 * time.Microsecond
```

Comment:
Does this pretty much mean sample everything? That's fine, just trying to understand the defaults (as we might need to update any "running juju in production" docs).

Reply:
100% we should re-address this before production. I've picked the defaults I wanted now, but they are not necessarily for production.